README.md

```diff
@@ -1,98 +1,49 @@
 
 ```
 As a marketeer
 So that I can see my name in lights
 I would like to register my name before playing an online game
-
+```
+```
 As a marketeer
 So that I can enjoy myself away from the daily grind
 I would like to be able to play rock/paper/scissors
 ```
 
-Hints on functionality
-
-- the marketeer should be able to enter their name before the game
-- the marketeer will be presented the choices (rock, paper and scissors)
-- the marketeer can choose one option
-- the game will choose a random option
-- a winner will be declared
-
-As usual please start by:
-
-* Forking this repo
-* Test-driving development of your app
-
-## Resources
+#### Getting Started ####
 
-* [HTML forms](https://www.w3schools.com/html/html_forms.asp)
-* [Capybara cheatsheet](https://devhints.io/capybara)
-* [Twitter bootstrap css library](https://getbootstrap.com/)
-* [Hosting on heroku](https://heroku.com)
-
-## Bonus level 1: Multiplayer
-
-Change the game so that two marketeers can play against each other ( _yes there are two of them_ ).
-
-## Bonus level 2: Rock, Paper, Scissors, Spock, Lizard
-
-Use the _special_ rules ( _you can find them here http://en.wikipedia.org/wiki/Rock-paper-scissors-lizard-Spock_)
-
-## Basic Rules
-
-- Rock beats Scissors
-- Scissors beats Paper
-- Paper beats Rock
-
-In code review we'll be hoping to see:
+Fork and then clone this repo using:
+```
+git clone [path-to-your-repo]
+```
 
-* All tests passing
-* High [Test coverage](https://github.com/makersacademy/course/blob/master/pills/test_coverage.md) (>95% is good)
-* The code is elegant: every class has a clear responsibility, methods are short etc.
-* Commits and short and scoped
+Install the necessary dependencies using:
+```
+bundle
+```
 
-Reviewers will potentially be using this [code review rubric](docs/review.md).  Referring to this rubric in advance may make the challenge somewhat easier.  You should be the judge of how much challenge you want this weekend.
+#### Usage ####
 
-## Notes on test coverage
+To start running the program, run:
+`rackup`
 
-Please ensure you have the following **AT THE TOP** of your `spec/spec_helper.rb` in order to have test coverage stats generated on your pull request:
+Then navigate to the following address in your browser:
+```
+localhost:9292
+```
 
-```ruby
-require 'simplecov'
-require 'simplecov-console'
+#### Running Tests ####
 
-SimpleCov.formatter = SimpleCov::Formatter::MultiFormatter.new([
-  SimpleCov::Formatter::Console,
-  # Want a nice code coverage website? Uncomment this next line!
-  # SimpleCov::Formatter::HTMLFormatter
-])
-SimpleCov.start
+To run the tests:
+```
+rspec
 ```
 
-You can see your test coverage when you run your tests. If you want this in a graphical form, uncomment the `HTMLFormatter` line and see what happens!
+#### Issues that I faced ####
+
+I found it difficult to test that when the user clicked a particular option, e.g. 'rock', that this changed the value of their choice instance variable.
```

Comment:
This is a really good practice of reviewing the challenge, leaving space to do further research and improve your code!